<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportProjectDialog</class>
 <widget class="QDialog" name="ExportProjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>724</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export Project</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="rightMargin">
      <number>10</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="m_add_button">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../../../../resources/egnim-images.qrc">
             <normaloff>:/images/32x32/add_file.png</normaloff>:/images/32x32/add_file.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="popupMode">
            <enum>QToolButton::InstantPopup</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="m_copy_button">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../../../../resources/egnim-images.qrc">
             <normaloff>:/images/32x32/copy_file.png</normaloff>:/images/32x32/copy_file.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="m_remove_button">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../../../../resources/egnim-images.qrc">
             <normaloff>:/images/32x32/remove_file.png</normaloff>:/images/32x32/remove_file.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonIconOnly</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListView" name="m_presets_list_view"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QStackedWidget" name="m_presets_stacked_widget"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_cancel_button">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_export_button">
       <property name="text">
        <string>Export</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_export_all_button">
       <property name="text">
        <string>Export All</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../resources/egnim-images.qrc"/>
 </resources>
 <connections/>
</ui>
